virtcontainers/clh.go

```diff
@@ -372,17 +364,12 @@ func (clh *cloudHypervisor) startSandbox(timeout int) error {
 	var strErr string
 	strErr, pid, err := clh.LaunchClh()
 	if err != nil {
-		return fmt.Errorf("failed to launch cloud-hypervisor: %s, error messages from log: %s", err, strErr)
-	}
-	clh.state.PID = pid
-
-	if err := clh.waitVMM(clhTimeout); err != nil {
```

Comment:
`waitVMM` is no longer required here?

Reply:
It was moved to [launchClh](https://github.com/kata-containers/runtime/pull/2648/files#diff-f1c6a53125ae38d7dbb5756ebea5771fR898) if fails to wait for the hypervisor, it read the output of the command and return it as part of the error output.